docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -978,6 +978,81 @@ If you want to use the server PacketFence to authenticate users on the Brocade s
 
 NOTE: Make sure to have a local account in case the switch can not reach the PacketFence server
 
+ICX 6600 Series
+^^^^^^^^^^^^^^^
+
+Those switches are supported using 802.1X for networks with or without VoIP. Tested on Firmware Version 08.0.60.
+
+* Global config settings:
+
+  aaa authentication dot1x default radius
+  aaa authorization coa enable
+  aaa accounting dot1x default start-stop radius
+  radius-server host 192.168.1.5
+  radius-server key useStrongerSecret
+  radius-server timeout 15
+  radius coa host 192.168.1.5 key useStrongerSecret
+
+* VLAN configuration (create all the VLANs necessary but only tag ports for VOIP VLAN, also note you must tag at least one port in each VLAN, the assumption is that the uplink interface is port 1/1/48, adjust as necessary):
+
+  vlan 1 name Access by port
+   tag ethe 1/1/48
+  vlan 2 name Registration by port
+   tag ethe 1/1/48
+  vlan 3 name Isolation by port
+   tag ethe 1/1/48
+  vlan 100 name Voice by port
+   tag ethe 1/1/1 to 1/1/48
+
+802.1x/MAC-Auth
++++++++++++++++
```

Comment:
Could you use new AsciiDoc syntax for [titles](https://packetfence.org/doc/PacketFence_Developers_Guide.html#_titles) ?